tests/jax/components/building/parameter_client_test.py

```diff
@@ -562,7 +562,7 @@ def test_trainer_parameter_client_separate_networks(
     assert mock_builder.store.trainer_parameter_client._get_call_counter == 0
     assert mock_builder.store.trainer_parameter_client._set_call_counter == 0
     assert mock_builder.store.trainer_parameter_client._set_get_call_counter == 0
-    assert mock_builder.store.trainer_parameter_client._update_period == 1
+    assert mock_builder.store.trainer_parameter_client._update_period == 5
```

Comment:
This assertion would fail if we change the default value of update_period in the config which is not ideal. We should rather set the value in the test and assert that it is the same as the set value

Reply:
Same above. we should just do what we do with executor update period in the test

Reply:
Agreed. I updated the tests to reflect this.